packages/network-graph/examples/network-graph/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "carbon-labs-network-graph-example",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "clean": "rimraf node_modules dist .cache",
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "rimraf": "^3.0.2",
+    "sass": "^1.55.0",
+    "vite": "^3.2.2"
+  },
+  "dependencies": {
+    "@carbon/styles": "^1.53.0",
+    "@carbon-labs/network-graph": "0.0.3"
```

changing this to always just pull in `latest` npm version

```suggestion
    "@carbon-labs/network-graph": "latest"
```